src/lang/english.txt

```diff
@@ -1690,6 +1690,9 @@ STR_CONFIG_SETTING_SCROLLMODE_LMB                               :Move map with L
 STR_CONFIG_SETTING_SMOOTH_SCROLLING                             :Smooth viewport scrolling: {STRING2}
 STR_CONFIG_SETTING_SMOOTH_SCROLLING_HELPTEXT                    :Control how the main view scrolls to a specific position when clicking on the smallmap or when issuing a command to scroll to a specific object on the map. If enabled, the viewport scrolls smoothly, if disabled it jumps directly to the targeted spot
 
+STR_CONFIG_SETTING_TOOLBAR_DROPDOWN_AUTOSELECT                  :Toolbar dropdowns automatically activate: {STRING2}
+STR_CONFIG_SETTING_TOOLBAR_DROPDOWN_AUTOSELECT_HELPTEXT         :Control whether the main toolbar dropdowns activate their default function when the mouse button is released. If not enabled then the dropdown menu will stay open so a selection can be made.
```

Comment:
```suggestion
STR_CONFIG_SETTING_TOOLBAR_DROPDOWN_AUTOSELECT_HELPTEXT         :Enable legacy main toolbar behavior. If enabled, clicking on a main toolbar button will activate its default function while holding the mouse button will open the dropdown, selecting whichever options is hovered over when released. If disabled, clicking on a main toolbar button will open the dropdown menu without needing to hold the mouse button.
```

Reply:
I think "legacy" is a bit strong, the current way is not being deprecated.

Reply:
Traditional?